prime-dotnet-webapi/Models/Agreements/AgreementVersionType.cs

```diff
@@ -0,0 +1,11 @@
+namespace Prime.Models
+{
+    public enum AgreementVersionType
+    {
+        CommunityPharmacistTOA = 1,
+        RegulatedUserTOA = 2,
+        OboTOA = 3,
```

Comment:
![image](https://user-images.githubusercontent.com/39168456/94748602-889c6400-0336-11eb-9075-1a0742f8aeb8.png)
🎵Well, OboToa hasn't always been this glam, I was a weekday MOA once🎵